performance_manager/lib/gtfs_static_table.py

```diff
@@ -0,0 +1,281 @@
+from typing import List, Optional
+from dataclasses import dataclass
+
+import logging
+import pandas
+import numpy
+
+import sqlalchemy as sa
+from sqlalchemy.orm import sessionmaker
+
+from .s3_utils import read_parquet, file_list_from_s3
+from .postgres_utils import MetadataLog, get_unprocessed_files
+from .postgres_schema import (
+    StaticFeedInfo,
+    StaticTrips,
+    StaticRoutes,
+    StaticStops,
+    StaticStopTimes,
+    StaticCalendar,
+)
+from .gtfs_utils import start_time_to_seconds
+
+
+@dataclass
+class StaticTableDetails:
+    """
+    Container class for loading static gtfs schedule tables
+    """
+
+    table_name: str
+    insert_table: str
+    columns_to_pull: List[str]
+    int64_cols: Optional[List[str]] = None
+    bool_cols: Optional[List[str]] = None
+    time_to_seconds_cols: Optional[List[str]] = None
+    data_table: pandas.DataFrame = pandas.DataFrame()
```

Comment:
Use `dataclass` for StaticTableDetails container class, each gtfs static schedule table will be represented by this object and flow through the ETL steps using this object.